src/_posts/platform/internals/2000-01-01-consumption-reports.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Consumption Reports
+modified_at: 2023-10-20 00:00:00
+tags: billing consumption report
+---
+
+## Your consumption report
+
+Scalingo lets you follow your consumption directly in your dashboard. Go to [https://dashboard.scalingo.com/consumption](https://dashboard.scalingo.com/consumption) and you'll see your consumption report for the current month.
+
+{% assign img_url = "https://cdn.scalingo.com/documentation/screenshot_dashboard_consumption_reports.png" %}
+{% include mdl_img.html %}
+
+{% note %}
+The amounts shown are frozen at the end of each billing period. An application transferred during the month is billed to its new owner, even if you can see its consumption for the period prior to its transfer.
+{% endnote %}
+
+## Current or past periods
+
+This report shows the consumption of all your applications for the current billing period and is updated daily at midnight UTC.
+
+When a billing period is over and the invoice is issued, you can explore your consumption to identify the main cost drivers.
+
+Please note that the consumption report document **is not an invoice**, but a breakdown of your consumption **excluding VAT**. It does not take into account the free trial period or any vouchers or credits.
```

Comment:
```suggestion
Please note that the consumption report document **is not an invoice**, but a breakdown of your consumption **excluding VAT**. It takes neither the free trial period nor any vouchers or credits into account .
```

Reply:
@Anouchka-M I have used the same translation as the one in the dashboard. Could you also make the change in the dashboard?